iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -204,7 +209,7 @@ cxx::expected<IpcChannelError> UnixDomainSocket::timedSend(const std::string& ms
                                                            const units::Duration& timeout) const noexcept
 {
     constexpr uint64_t NULL_TERMINATOR_SIZE = 1U;
-    if (msg.size() > m_maxMessageSize + NULL_TERMINATOR_SIZE)
+    if (msg.size() >= m_maxMessageSize + NULL_TERMINATOR_SIZE)
```

Comment:
Hmm, shouldn't this be just `msg.size > m_maxMessageSize`?

Reply:
You are right, but I also have to reduce the `MAX_MESSAGE_SIZE` by the terminator size since it is always added.